src/components/Card/index.ts

```diff
@@ -1,5 +1,16 @@
+import { Actions as BottomActions } from "./Actions";
```

Comment:
suggestion: I'd suggest leave Action, as Action can be place anywhere but `BottomActions` implies something wrong

Reply:
Does it really imply something's wrong? My intention was to show where the actions should be placed and to differentiate them from Toolbar.